### DISCUSSION TITLE
This is frustrating...

### DISCUSSION THREAD

**Discussion by babakfp** [pnpm/pnpm#8677] [General]
Create an empty folder.
Run `pnpm add express`, but nothing happens.
I expected a `package.json` file to be created, the dependency to be added, and the package installed.
Okay, so you create the `package.json` file manually. Now, installing a package should work for sure, right? Wrong!
You get an error: `Unexpected end of JSON input while parsing empty string in ...\package.json`.
Now, you need to add `{}` to the `package.json` for it to work.
This is frustrating.

**Comment by xty** [comment]
If you haven't initialized your project, try running `pnpm init` in your project directory before adding dependencies.

**Reply by babakfp** [reply]
Thanks, but:
I expect the `add` command to work as I explained above.
I don't want all the bloat that comes with the `init` command.
Even if the `init` command wasn't bloated, I would still expect the `add` command to work.
